.github/workflows/release.yml

```diff
@@ -96,3 +96,15 @@ jobs:
           echo "${LAUNCHPAD_DEPLOY_KEY}" | gpg --import
           echo "trusted-key ${LAUNCHPAD_UPLOADER_PGP_KEY}" > ~/.gnupg/gpg.conf
           rake -C packages ubuntu
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ruby
+      - name: Post Announcement
+        run: |
+          bundle install
```

Comment:
Can we use `bundler-cache: true` with `setup-ruby`?

Reply:
fix: [7f4be7b](https://github.com/groonga/groonga/pull/2294/commits/7f4be7b4a73188275f1589c3bec42e9c14959546) Yes, we can use it!

Reply:
Do we still need `bundle install` with `bundler-cache: true`?

Reply:
fix: [0fc5908](https://github.com/groonga/groonga/pull/2294/commits/0fc590843ca063528e3815ec66fb4cfeaaad93a9) We don't need. I misunderstood the `bundler-cache` option.
- https://github.com/ruby/setup-ruby?tab=readme-ov-file#caching-bundle-install-automatically